docs/components/attention/Example.vue

```diff
@@ -85,7 +84,7 @@ const popoverIconTargetShowing = ref(false)
       </w-attention>
     </div>
     <div>
-      <h4>Popover with icon as target element</h4>
+      <h3 class="t4">Popover with icon as target element</h3>
       <w-button
         :aria-expanded="popoverIconTargetShowing"
         aria-controls="popover-icon-target-example"
```

Comment:
Not sure what has caused this, but in prod this icon is not clipped on the right side, like it is on this branch
![Screenshot 2024-06-10 at 14 51 11](https://github.com/warp-ds/tech-docs/assets/41303231/2a9073da-5575-4002-9b8c-c0c5ebbce47e)

Reply:
? 🤷 
![image](https://github.com/warp-ds/tech-docs/assets/7531505/ae707cb2-3cce-4f79-8d67-9b352015c680)

Reply:
Whoa, I restarted Chrome and it's correct now 😄